crates/sui-adapter/src/adapter.rs

```diff
@@ -234,7 +234,7 @@ pub fn publish<
     ctx: &mut TxContext,
     gas_status: &mut SuiGasStatus,
 ) -> Result<(), ExecutionError> {
-    gas_status.charge_publish_package(module_bytes.iter().map(|v| v.len()).sum())?;
+    gas_status.charge_publish_package()?;
```

Comment:
Do we not also want to charge for the transaction size? or is that done somewhere else?